pkg/controller/node/nodecontroller.go

```diff
@@ -261,16 +290,18 @@ func (nc *NodeController) Run(period time.Duration) {
 	}, nodeEvictionPeriod, util.NeverStop)
 }
 
-// Generates num pod CIDRs that could be assigned to nodes.
-func generateCIDRs(clusterCIDR *net.IPNet, num int) sets.String {
-	res := sets.NewString()
+// translateCIDRs does translating pod CIDRs index to pod CIDRs that could be
```

Comment:
translateCIDRs translates pod CIDR index to the CIDR that could be assigned to the node.